plugins/modem-manager/fu-mm-device.c

```diff
@@ -1109,6 +1109,8 @@ fu_mm_device_write_firmware_mbim_qdu (FuDevice *device, GBytes *fw, GError **err
 		return FALSE;
 
 	fu_device_set_status (device, FWUPD_STATUS_DEVICE_READ);
+	fu_device_set_remove_delay (device, MAX_WAIT_TIME_SECS * 1000);
+	fu_device_add_flag (device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
```

Comment:
we've changed out style guide -- can you remove the spaces before the brackets on these two added lines please and then precommit will be happy. Then LGTM, thanks.

Reply:
ok, thank you for reminding me .

Reply:
Would suggest you run contrib/setup on your side to set up pre-commit rules for the future.